packages/augur-ui/src/modules/market/components/market-header/market-header-reporting.tsx

```diff
@@ -7,66 +7,63 @@ import MarketLink from "modules/market/components/market-link/market-link";
 import {
   TYPE_DISPUTE,
   TYPE_REPORT,
-  MARKET_STATUS_MESSAGES,
   REPORTING_STATE,
 } from "modules/common/constants";
-import {
-  CountdownProgress,
-  formatTime
-} from "modules/common/progress";
 import { PrimaryButton } from "modules/common/buttons";
-
-import canClaimProceeds from "utils/can-claim-proceeds";
+import { createBigNumber } from "utils/create-big-number";
+import { ZERO } from 'modules/common/constants';
 
 export default class MarketHeaderReporting extends Component {
   static propTypes = {
-    currentTimestamp: PropTypes.number.isRequired,
     market: PropTypes.object.isRequired,
     isDesignatedReporter: PropTypes.bool,
-    finalizeMarket: PropTypes.func.isRequired,
-    claimTradingProceeds: PropTypes.func.isRequired,
+    claimMarketsProceeds: PropTypes.func.isRequired,
     tentativeWinner: PropTypes.object,
     isLogged: PropTypes.bool,
-    location: PropTypes.object.isRequired
+    location: PropTypes.object.isRequired,
+    accountPositions: PropTypes.object.isRequired,
   };
 
   static defaultProps = {
     isDesignatedReporter: false,
     tentativeWinner: {},
-    isLogged: false
+    isLogged: false,
   };
 
   constructor(props) {
     super(props);
     this.state = {
-      disableFinalize: false
+      disableFinalize: false,
     };
   }
 
   render() {
     const {
       market,
       isDesignatedReporter,
-      finalizeMarket,
-      claimTradingProceeds,
+      claimMarketsProceeds,
       tentativeWinner,
       isLogged,
-      currentTimestamp,
-      location
+      location,
+      accountPositions
     } = this.props;
     const {
       reportingState,
       id,
       consensus,
-      outstandingReturns,
-      finalizationTime
+      finalizationTime,
     } = market;
 
-    const canClaim = canClaimProceeds(
-      finalizationTime,
-      outstandingReturns,
-      currentTimestamp
-    );
+    let canClaim = false;
```

Comment:
There are two instances of calculating canClaim. It doesnt need to be in the render method here. Pull it out into a helper and call that helper from containers and pass in a canClaim prop